packages/docs/src/components/ShowcaseCard.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import {
+  Box,
+  Card,
+  CardActions,
+  CardContent,
+  Button,
+  Typography,
+  CardActionArea,
+  Chip,
+  Stack
+} from '@mui/material';
+import { ShowcaseItem } from "../data/showcase";
+
+
+export const ShowcaseCard = ({ data }: { data: ShowcaseItem }) => {
+  return (
+    <Card 
+      sx={{
+        flex: 1,
+        '& .MuiCardActionArea-root:hover': {
+          color: 'inherit',
+          textDecoration: 'none'
+        }
+      }}
+      variant="outlined">
```

Comment:
```suggestion
      variant="outlined"
      sx={{
        flex: 1,
        '& .MuiCardActionArea-root:hover': {
          color: 'inherit',
          textDecoration: 'none'
        }
      }}
    >
```

Reply:
suggestion: Add in a placeholder image along the lines of Rich's comment on discord